src/components/Section.js

```diff
@@ -1,17 +1,20 @@
-export default class Section {
-  constructor({ items, renderer }, containerSelector) {
+class Section {
+  constructor({ items, renderer }, cardElementsSelector) {
     this._items = items;
     this._renderer = renderer;
-    this._container = document.querySelector(containerSelector);
+    this._cardElementsContainer = document.querySelector(cardElementsSelector);
```

Comment:
Needs correction: `Section` is a general class. It can render any kind of items, not only cards. Please, rename the variables to `_container` and `selector` (remove cards mentions)